enum.go

```diff
@@ -112,10 +112,33 @@ func AllConnectiveEnum() []string {
 	}
 }
 
+// ContactType represents the method of contact.
+type ServiceOwnershipCheckContactType string
+
+const (
+	ServiceOwnershipCheckContactTypeAny         ServiceOwnershipCheckContactType = "any"          // A Slack channel contact method.
```

Comment:
`ANY` was the main reason this needed to be its own type. Its a valid enum value for the service ownership check but not for a regular contact type.